package.json

```diff
@@ -28,7 +28,9 @@
   },
   "dependencies": {
     "dotenv": "^16.0.0",
-    "pg": "^8.7.3"
+    "express": "^4.18.2",
+    "pg": "^8.11.3",
+    "supertest": "^6.3.3"
```

Comment:
Is this a dependency or a dev dependency? Will we need it when hosting to production?